streams/bo.m3u

```diff
@@ -45,3 +53,11 @@ http://136.243.3.70:1935/TvUniversitaria/TvUniversitaria/playlist.m3u8
 https://solo.disfrutaenlared.com:1936/tuch/ngrp:tuch_all/playlist.m3u8
 #EXTINF:-1 tvg-id="XTOTV.bo" status="online",XTOTV (404p) [Not 24/7]
 http://www.channel.tevemi.com:1935/XtoTv/XtoTv/playlist.m3u8
+#EXTINF:-1 tvg-id="UPPTV.bo" status="online",UPP TV (480p) [Not 24/7]
+https://d2byzrnrdb259r.cloudfront.net/hls/upptv.m3u8
+#EXTINF:-1 tvg-id="UnitelGlobal.bo" status="online",Unitel Global
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7vsezg
```

Comment:
down

Reply:
not live atm

Reply:
![image](https://user-images.githubusercontent.com/30985701/188329611-8de95df8-ad0c-4151-aa30-8f23c4a30c1a.png)